timesketch/api/v1/resources/sigma.py

```diff
@@ -58,17 +88,15 @@ def get(self):
         meta = {"current_user": current_user.username, "rules_count": len(sigma_rules)}
         return jsonify({"objects": sigma_rules, "meta": meta})
 
-
+# TODO(jaegeral): deprecate this class
```

Comment:
Well that will not be deprecated in this PR, so I rather have this TODO. It will be deprecated once the API client is also ported over to the newly created API methis in here.